### PR TITLE
[FIX] Unable to close chats when comments is disabled

### PR DESCRIPTION
**Pull request by murtaza98**
## Proposed changes (including videos or screenshots)

Fixes https://github.com/RocketChat/Rocket.Chat/issues/25954

### CODE REVIEW

**Comment by KevLehman** [comment]
This is not a regression per se, but this would be a really annoying issue 😬 to keep on 5.0. 

@ggazzo , agree on put this as "regression" (it will go through normal QA anyways, but since the fix is pretty self contained, we should be good)

**Comment by murtaza98** [comment]
Yeah, I agree @KevLehman. And considering the release candidate deadline was extended we should have some time to get it reviewed and qa'd.

File: apps/meteor/client/components/Omnichannel/modals/CloseChatModal.tsx [hunk 1]
```diff
@@ -113,7 +117,7 @@ const CloseChatModal = ({
 		<GenericModal
 			variant='warning'
 			title={t('Are_you_sure_you_want_to_close_this_chat')}
-			onConfirm={onConfirm}
+			onConfirm={onSubmitWithoutCommentsAndTags}
```
Comment (MartinSchoeler) [inline apps/meteor/client/components/Omnichannel/modals/CloseChatModal.tsx:120]:
```suggestion
			onConfirm={(): void => onConfirm()}
```

File: apps/meteor/client/components/Omnichannel/modals/CloseChatModal.tsx [hunk 2]
```diff
@@ -61,6 +61,10 @@ const CloseChatModal = ({
 		return cannotSendTag || cannotSendComment;
 	}, [comment, commentRequired, errors, tagRequired, tags]);
 
+	const onSubmitWithoutCommentsAndTags = useCallback((): void => {
+		onConfirm();
+	}, [onConfirm]);
+
```
Comment (MartinSchoeler) [inline apps/meteor/client/components/Omnichannel/modals/CloseChatModal.tsx:67]:
```suggestion
	
```

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]
No need to add a new callback just for this, I think it would be better to do it inline

File: apps/meteor/client/components/Omnichannel/modals/CloseChatModal.tsx [hunk 3]
```diff
@@ -61,6 +61,7 @@ const CloseChatModal = ({
 		return cannotSendTag || cannotSendComment;
 	}, [comment, commentRequired, errors, tagRequired, tags]);
 
+
```
Comment (murtaza98) [inline apps/meteor/client/components/Omnichannel/modals/CloseChatModal.tsx:64]:
```suggestion
```

**Comment by murtaza98** [comment]
> No need to add a new callback just for this, I think it would be better to do it inline

Done :heavy_check_mark: , please check

**Review by aleksandernsilva — APPROVED** [review]

**Pull request merged**